### PR TITLE
feat: action reports view

### LINKED ISSUE #79: Reports view
## Feature Request

Create simple reports view with blocks showing numbers.

Possible data to show:

- Number of actions closed today
- A number of actions that were not closed today.
- A number of actions in progress.

### PR DESCRIPTION
**Pull request by machi1990**
Fix #79

##### Checklist

- [ ] `npm test` passes
- [ ] `npm run build` works
- [ ] tests are included
- [ ] documentation is changed or added

### CODE REVIEW

**Comment by machi1990** [comment]
Screenshot 2020-05-29 at 13 43 50

**Comment by wtrocki** [comment]
Absolute masterpiece!

File: platform/client-admin/src/pages/ActionReportPage.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    reportsRows.push(entries.slice(row, row + REPORT_COLUMN_SIZE));
+  }
+
+  const gridContent = reportsRows.map((reportRow, index) => {
+    const columns = reportRow.map((column) => {
+      return <IonCol key={column[0]}>
+        <div>
+          {humanizeString(column[0])}
+          <br/>
+          {column[1].items.length}
```
Comment (wtrocki) [inline platform/client-admin/src/pages/ActionReportPage.tsx:55]: Ok.. This is kinda badish and shows that aggregations are sometimes useful. We will have count implemented soon so hopefully that would do the job

Comment (machi1990) [reply platform/client-admin/src/pages/ActionReportPage.tsx:55]: Yeap. Would be very useful. 

I had this crazy idea about implementing a `crud-stats-schema-codegen` plugin but decided against it.

**Comment by machi1990** [comment]
> Absolute masterpiece! I will add headers - today and total. today stats on the tops. 

You mean a view's header? `Today's statistics and total actions` something like that? 

> I will also add incidents number in spepare pr - actions that were not delivered etc.

What'll be the business logic? An action is not delivered if it took more than a day to be completed? (Assigned/created and it has been more than a day?)

File: platform/client-admin/src/pages/ActionReportPage.tsx (hunk 1 above)
Comment (wtrocki) [reply platform/client-admin/src/pages/ActionReportPage.tsx:55]: We have count available already in schema but it is not implemented

**Comment by wtrocki** [comment]
Ah.. I get you now. Let it be this way

**Comment by machi1990** [comment]
Merging this and can be refined later.

**Pull request merged**